pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1099,68 +1099,51 @@ func buildGatewayListener(opts gatewayListenerOpts, transport istionetworking.Tr
 		})
 	}
 
-	var res *listener.Listener
+	res := &listener.Listener{
+		// TODO: need to sanitize the opts.bind if its a UDS socket, as it could have colons, that envoy doesn't like
+		Address:          util.BuildAddress(opts.bind, uint32(opts.port)),
```

Comment:
This lost the QUIC part? `util.BuildNetworkAddress(opts.bind, uint32(opts.port), istionetworking.TransportProtocolQUIC),`

Reply:
Nice catch